src/common.cu

```diff
@@ -101,13 +110,13 @@ static int enable_out_of_place = 1;
 static double parsesize(const char *value) {
     long long int units;
     double size;
-    char size_lit;
+    char size_lit[2];
```

Comment:
why do we need to change size_lit?

Reply:
it's because of compile warning from the code below
int count = sscanf(value, "%lf %1s", &size, size_lit);